<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pixel Information</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>10</number>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>15</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>X:</string>
         </property>
         <property name="buddy" >
          <cstring>xSpin</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="xSpin" >
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols" >
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="accelerated" >
          <bool>true</bool>
         </property>
         <property name="correctionMode" >
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="keyboardTracking" >
          <bool>false</bool>
         </property>
         <property name="maximum" >
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Y:</string>
         </property>
         <property name="buddy" >
          <cstring>ySpin</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ySpin" >
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="accelerated" >
          <bool>true</bool>
         </property>
         <property name="correctionMode" >
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="keyboardTracking" >
          <bool>false</bool>
         </property>
         <property name="maximum" >
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_20" >
       <property name="text" >
        <string>Scanline order:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QRadioButton" name="topDownRadio" >
         <property name="text" >
          <string>&amp;Top-Down</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="bottomUpRadio" >
         <property name="text" >
          <string>&amp;Bottom-Up</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Displayed:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="spacing" >
        <number>2</number>
       </property>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLabel" name="rDispLabel" >
         <property name="text" >
          <string>r</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>G:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLabel" name="gDispLabel" >
         <property name="text" >
          <string>g</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLabel" name="bDispLabel" >
         <property name="text" >
          <string>b</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>15</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13" >
       <property name="text" >
        <string>Stored:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="spacing" >
        <number>2</number>
       </property>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_19" >
         <property name="text" >
          <string>R:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLabel" name="rLabel" >
         <property name="text" >
          <string>r</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="label_17" >
         <property name="text" >
          <string>G:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLabel" name="gLabel" >
         <property name="text" >
          <string>g</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="label_14" >
         <property name="text" >
          <string>B:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLabel" name="bLabel" >
         <property name="text" >
          <string>b</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>15</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
